sci-mathematics/octave/octave-9.2.0.ebuild

```diff
@@ -110,36 +97,13 @@ BDEPEND="
 		dev-texlive/texlive-metapost
 		virtual/latex-base
 	)
-	qt5? ( dev-qt/linguist-tools:5 )
 	qrupdate? ( app-misc/pax-utils )
 	sparse? ( app-misc/pax-utils )
 "
 
-# There are three ways to plot in Octave:
-#
-#   1. The old gnuplot renderer
-#   2. The OpenGL renderer using the FLTK backend
-#   3. The OpenGL renderer using the Qt backend
-#
-# It's possible to use the Qt GUI without OpenGL, but OpenGL rendering
-# is all that FLTK is used for, so it doesn't make sense to enable
-# USE=fltk without USE=opengl.
-#
-# Building without either USE=gnuplot or USE=opengl is technically legal,
-# but will leave you unable to plot anything.
-#
-# Octave's FLTK support is unofficially deprecated, in the sense that
-# you'll often get "why are you using FLTK?" in response to
-# bugs. (Upstream bug 59321 for a random example.) In the future, it
-# will probably make sense to merge USE=opengl and USE=X into USE=qt[56],
```

Comment:
Let's drop the opengl and X flags as suggested here. So enabling `USE=gui` would also enable the opengl and X configure options and dependencies.